SQL_testing.php

```diff
@@ -0,0 +1,50 @@
+<?php
+	$title_Max_Length = 128;
+	$lecturer_Max_Length = 128;
+	$description_Max_Length = 1024;
```

Comment:
I like that you extract these as variables - it's easier to change and maintain.